docs/blog/posts/2022-10-10-work_with_large_zip_files_in_python.md

```diff
@@ -0,0 +1,136 @@
+---
+date:
+  created: 2022-10-10
+  updated: 2024-11-19
+categories:
+    - zip
+authors:
+    - kevin128
```

Comment:
I think Jeff wrote it right

Reply:
I used the same example we used before. Please let me know which example you would like me to use?

Reply:
Or can @Jeffotter , can you upload those old files to that directory?

Reply:
Is this something we want being recreated? Or is this a demonstration of how to read this in general

Reply:
don't matter to me. Was just noting my thoughts